blenderproc/command_line.py

```diff
@@ -11,8 +10,9 @@
 sys.path.append(repo_root_directory)
 
 # pylint: disable=wrong-import-position
-from blenderproc.python.utility.SetupUtility import SetupUtility
+from blenderproc.python.utility.SetupUtility import SetupUtility, is_using_external_bpy_module
 from blenderproc.python.utility.InstallUtility import InstallUtility
+from blenderproc.python.utility.Utility import Utility
```

Comment:
Utility should not be imported here, as it needs bpy, which might not be available in the users env. Maybe put the `clean_temp_dir()` into SetupUtility?

Reply:
Fixed.